cluster-autoscaler/FAQ.md

```diff
@@ -933,7 +934,9 @@ Events:
 ```
 ### My cluster is below minimum / above maximum number of nodes, but CA did not fix that! Why?
 
-Cluster Autoscaler will not scale the cluster beyond these limits, but does not enforce them. If your cluster is below the minimum number of nodes configured for Cluster Autoscaler, it will be scaled up *only* in presence of unschedulable pods.
+Cluster Autoscaler will not scale the cluster beyond these limits, but does not enforce them. By default, if your cluster is below the minimum number of nodes configured for Cluster Autoscaler, it will be scaled up *only* in presence of unschedulable pods.
+
+Recently, a new flag `--scale-up-to-node-group-min-size-enabled` was introduced to support this scenario. CA will scale up the node groups that have less nodes than the configured minimum size to the node group minimum size. To enable this feature, please set it to `true` in the command.
```

Comment:
Let's avoid words that will quickly get outdated. How about replacing "Recently" with "Starting with CA 1.26.0"?

Reply:
Good point. Let's target 1.26.0.